src/ui-shell/sidenav/sidenav-menu.component.ts

```diff
@@ -113,10 +113,6 @@ export class SideNavMenu implements AfterContentInit, OnDestroy {
 	}
 
 	protected findActiveChildren() {
-		if (this.sidenavItems.some(item => item.active)) {
-			this.hasActiveChild = true;
-		} else {
-			this.hasActiveChild = false;
-		}
```

Comment:
Nice catch!

Reply:
Thanks :)
I was looking for how things work when I stumbled upon this block :D